src/services/infrastructure/entity-resolver/room.resolver.service.ts

```
@@ -25,7 +25,7 @@ export class RoomResolverService {
     private entityManager: EntityManager,
     @Inject(WINSTON_MODULE_NEST_PROVIDER)
     private readonly logger: LoggerService
-  ) {}
+  ) { }
```

ESLint

[error] Delete `·`

(prettier/prettier)